SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Report a Vulnerability
+
+Please report security issues or vulnerabilities to the [Intel® Security Center].
+
+For more information on how Intel® works to resolve security issues, see
```

Comment:
```suggestion
For more information on how Intel® resolves security issues, see
```

Reply:
I'm not sure if we can change the content here. @maria-Petrova It's true?

Reply:
if it's a wording provided by legal, then it's ok to leave as it is

Reply:
Yes, the wording was provided by legal, so please leave it the way it is.